### DISCUSSION TITLE
Spurious notification for Discussion I was not actually mentioned in

### DISCUSSION THREAD

**Discussion by BGR360** [community/community#43143] [Discussions] [answered]
### Select Topic Area

Bug

### Body

Today I saw the following notification for [this discussion](https://github.com/blakeblackshear/frigate/discussions/4793) on a repository I have never seen before:

<img width="1144" alt="image" src="https://user-images.githubusercontent.com/13339928/210267559-03d5a22a-49d7-4596-afb5-bac8e06a28f2.png">

Upon inspecting the discussion, I find my username nowhere to be found. How did GH detect that I was "mentioned" here?

cc @euDominic

**Comment by airtower-luna** [comment, marked as answer]
A few days ago there was a spam wave with lots of mentions in addition to spam links. Maybe you got targeted in that? It seems like when the spam posts got deleted the notifications they triggered stayed. :sweat_smile:

If you got a notification email you can look at it to confirm if it was that.

**Reply by jricciardi** [reply]
Marking this as the answer! In this case, you likely were @ mentioned by a spam campaign and our system has removed the spammy mention & took measures against the spammers.

**Reply by CzarScar** [reply]
What should I do if I still get the notification? It seems I have been targeted by the digital mining.
<img width="884" alt="image" src="https://github.com/community/community/assets/5074190/4b09d4a3-26de-48f3-9cdc-5e1fab4ade73">

**Comment by jgonggrijp** [comment]
I seem to have been hit by this as well. Notification for a PR in a repo I never heard of, supposedly because I was mentioned, but mention nowhere to be found.

![Schermafbeelding 2024-04-27 om 23 15 16](https://github.com/community/community/assets/1540185/0028fb72-8338-4d48-ae65-130db6cdef3d)

https://github.com/Pure-Peace/nan-lang/pull/1?notification_referrer_id=NT_kwDOABeAWbI2NDM3NDMzNjI5OjE1NDAxODU&notifications_query=reason%3Aparticipating#issuecomment-2080384531

It is great that you clean up the spam, but could you also remove the notifications? I spent a lot of time first trying to find where I was mentioned and then searching GH support. If this is spam, then a lot of people are potentially spending so much time.
